### DISCUSSION TITLE
Different output dir for scheduled mass scans

### DISCUSSION THREAD

**Discussion by Steve-Brule** [cross-seed/cross-seed#343] [Q&A]
Is it possible to have cross-seed output files to a different folder for scheduled scans?

**Comment by mmgoodnow** [comment]
what are you trying to achieve with this?

**Reply by DoAndroids** [reply]
Hi

Similar request from me (i.e. would like a different output dir in some cases).

Scenario: I want to use autobrr to push notifications to the cross-seed service, which will determine if a match is found - and for cross-seed to write/save the .torrent file to "watchCrossSeedTV" or "watchCrossSeedMovies" directories, depending on what I'm pushing.

The request is essentially the same I believe (i.e. have an option that allows an override of the "outputDir" value.

All the best

**Reply by mmgoodnow** [reply]
Why does the output dir need to be different? I ask because this would be incompatible with RSS. I want to make sure I fully understand the problem so I can solve for more than just one user.

**Reply by DoAndroids** [reply]
So, in my case the client is rTorrent. Direct injection doesn't work correctly (https://github.com/cross-seed/cross-seed/discussions/339).
Therefore, I write torrent files out to watch directories instead.

I have multiple watch directories set up in the rtorrent.rc file, because I have Plex running on there and there are conflicts in identifying 'movies' vs. 'TV shows' - so I separate these on the filesystem.

```
schedule = watch_directory_3,10,10,"load.normal=/rtorrent/watchCrossSeed/*.torrent,d.directory.set=/media/movies,d.set_custom1=CrossS
eed,d.complete=1,d.delete_tied="
schedule = watch_directory_4,10,10,"load.normal=/rtorrent/watchCrossSeedTV/*.torrent,d.directory.set=/media/tvshows,d.set_custom1=Cro
ssSeed,d.complete=1,d.delete_tied="
```

So, the movies get written to "watchCrossSeed" and TV shows to "watchCrossSeedTV".
Therefore, the save directories get applied and labels set to "CrossSeed" when the torrents are loaded by the client.
I have a Python script that connects to the scgi socket and does a force recheck then starts (assuming completion & a paused state).
